numpy/random/pcg32.pyx

```diff
@@ -248,24 +248,29 @@ cdef class PCG32:
     cdef jump_inplace(self, iter):
         """
         Jump state in-place
-
+        
         Not part of public API
-
+        
```

Comment:
These blank lines have trailing spaces

Reply:
Removed, and a couple of bonus trailing whitespaces